packages/paste-core/components/button-group/__tests__/index.spec.tsx

```diff
@@ -22,18 +22,6 @@ describe('ButtonGroup', () => {
     }
   });
 
-  it('should accept attached prop and apply styles', () => {
```

Comment:
Note: I ended up removing these tests because they are tested by Chromatic and because jest was having issues with the nested styles for the middle button in the group.